gateway-incoming.yaml

```diff
@@ -109,7 +110,14 @@ paths:
             }
   /sync:
     post:
-      description: "Sync the data from Device42 into the IPAM Facade database"
+      description: "Synchronize the IPAM data from Device42 with the IPAM Facade database"
```

Comment:
Task: I can't comment on it because it's not included in this PR, but the `lambda` section for this API has a hard coded empty request body, which I think will overwrite whatever is passed.